src/catalog/rel_table_schema.cpp

```diff
@@ -60,5 +66,18 @@ std::unique_ptr<RelTableSchema> RelTableSchema::deserialize(Deserializer& deseri
     return schema;
 }
 
+std::string RelTableSchema::toCypher(const Catalog* catalog) const {
+    std::stringstream ss;
+    auto srcTableName = catalog->getTableName(&DUMMY_READ_TRANSACTION, srcTableID);
```

Comment:
This makes mean think you might wanna pass `ClientContext` instead of `Catalog` in this function because `ClientContext` can give you the proper transaction.